com.unity.shadergraph/Editor/Generation/Targets/BuiltIn/ShaderLibrary/Shim/HLSLSupportShim.hlsl

```diff
@@ -3,6 +3,13 @@
 
 #define HLSL_SUPPORT_INCLUDED
 
+#if !defined(SHADER_API_GLES)
```

Comment:
All changes in this file were copied out of HLSLSupport.cginc and were necessary defines to remove the duplicate files.